architecture/core_concepts/deployments.adoc

```diff
@@ -70,6 +70,63 @@ spec:
 <4> Labels on the pod should include those from the label selector.
 <5> The maximum name length after expanding any parameters is 63 characters.
 
+[[replica-set]]
+== Replica set
+Similar to a xref:replication-controllers[replication controller], a replica set
+ensures that a specified number of pod replicas are running at any given time.
+The difference between a replica set and a replication controller is that a
+replica set supports set-based selector requirements whereas a replication
+controller only supports equality-based selector requirements.
+
+[NOTE]
+====
+It is recommended to use
+xref:deployments-and-deployment-configurations[Deployments] rather than using
+replica sets. Replica sets can be used independently, but it is used by
+deployments to orchestrate pod creation, deletion and updates. When you use
+deployments you do not have to manually manage the replica sets that they
+create. Deployments own and manage their replica sets automatically and provides
+declarative updates to pods. Only use replica sets if you require custom update
+orchestration or do not require updates at all.
```

Comment:
I think this might be burying the lede. Maybe something like:

----
Only use replica sets if you require custom update orchestration or do not require updates at all, otherwise, use xref:deployments-and-deployment- configurations[Deployments]. Replica sets can be used independently, but are used by deployments to orchestrate pod creation, deletion, and updates. Deployments manage their replica sets automatically, provide declarative updates to pods, and do not have to manually manage the replica sets that they create.
----

Note the grammar and punctuation changes to the sentences as well.